CONTRIBUTING.md

```diff
@@ -70,14 +70,14 @@ is to use community discussion to improve proposals and assist authors.
 
 ### Help contribute ideas to Carbon
 
-If you have ideas for Carbon, we encourage you to discuss it with the community,
+If you have ideas for Carbon, we encourage you to discuss them with the community,
 and potentially prepare a proposal for it. Ultimately, any changes or
 improvements to Carbon will need to turn into a proposal and go through our
 [evolution process](docs/project/evolution.md).
 
 If you do start working on a proposal, keep in mind that this requires a time
 investment to discuss the idea with the community, get it reviewed, and
-eventually implemented. A good starting point is to read through the
+eventually implemented it. A good starting point is to read through the
```

Comment:
I don't agree with this edit.  I actually think the error is the `,` before the and.

Reply:
I think it's just a clumsy sentence. Probably a better phrasing would be:

to get an idea discussed with the community, reviewed, and eventually implemented.

Reply:
I like that phrasing better.  Do you mind using the one suggested by jonmeow?